tests/it/io/avro/read.rs

```diff
@@ -41,13 +40,6 @@ fn schema() -> (AvroSchema, Schema) {
                 "type": "enum",
                 "name": "",
                 "symbols" : ["SPADES", "HEARTS", "DIAMONDS", "CLUBS"]
-            }},
-            {"name": "duration",
-             "logicalType": "duration",
```

Comment:
this is deleted because this is an invalid schema (which `avro-rs` consumes and incorrectly writes an avro file with).